Makefile

```diff
@@ -453,8 +453,14 @@ CPPCHECK_OPT+=--suppress=preprocessorErrorDirective
 EXTRA_C_SRCS:=$(wildcard uctest/*.c)
 EXTRA_SRCS:=$(wildcard $(foreach n,sample utest uctest,$n/*.cpp $n/*.h))
 EXTRA_SRCS+=$(EXTRA_C_SRCS)
+
+EXTRA_SRCS+=$(wildcard jclklib/common/*.c*)
+EXTRA_SRCS+=$(wildcard jclklib/proxy/*.c*)
+EXTRA_SRCS+=$(wildcard jclklib/client/*.c*)
+EXTRA_SRCS+=$(wildcard jclklib/sample/*.c*)
```

Comment:
i think no need to add sample, as sample should not compile together, like checksync app

Reply:
this is just for format checking. i think erez do check the format for his sample. so i just follow ? 
EXTRA_SRCS:=$(wildcard $(foreach n,sample utest uctest,$n/*.cpp $n/*.h))
RCS)

Reply:
oh i see. only for format check.